control/Controller.php

```diff
@@ -127,10 +127,11 @@ public function doInit() {
 	/**
 	 * Returns a link to this controller. Overload with your own Link rules if they exist.
 	 *
+	 * @param string $action Optional action
 	 * @return string
 	 */
-	public function Link() {
-		return get_class($this) .'/';
+	public function Link($action = null) {
+		return Controller::join_links(get_class($this), $action, '/');
```

Comment:
1) Why not just call with get_class + `func_get_args` - $action isn't guaranteed to be an action, it'll just be whatever the caller wants
2) Does it need to finish with a '/'? And if so, can't join_links do that, instead of added it here?